support-frontend/assets/pages/showcase/showcase.scss

```diff
@@ -113,19 +113,23 @@
         }
       }
 
-      @include mq($from: tablet, $until: desktop) {
+      @include mq($from: tablet, $until: leftCol) {
+        .component-text {
+          max-width: 75%;
+        }
+
         .component-content__image {
           right: -200px;
         }
-
+        
```

Comment:
not sure which editor you're using but watch out for stray white space changes as they cause noise in the diffs.  I think IntelliJ automatically strips out trailing whitespace as soon as you edit a file, not sure if that's something you have to enable in other editors.

Reply:
We use `eslint` which can be used to auto-fix code when it's run with the `--fix` flag. We have a validation script in our `package.json` that can be run via the command `yarn validate` on the command line, this calls `eslint` with the `--fix` flag, this should hopefully tidy up whitespace.

Reply:
Looks like Eslint only checks/validates the .js and .jsx files and not the .scss files.

Reply:
completely missed this was a `.scss` file 😆